packages/react-components/src/organisms/CalendarList.tsx

```diff
@@ -54,67 +70,113 @@ const orderList = css({
     borderTop: `1px solid ${steel.rgb}`,
   },
 });
+
+const showMoreStyles = css({
+  display: 'flex',
+  justifyContent: 'center',
+  paddingTop: `${15 / perRem}em`,
+  paddingBottom: `${15 / perRem}em`,
+  borderTop: `1px solid ${steel.rgb}`,
+});
+
 interface CalendarListProps {
   calendars: ReadonlyArray<CalendarResponse>;
-  page: 'calendar' | 'event' | 'group';
+  page: 'calendar' | 'event' | 'group' | 'calendar-working-group';
+  showBottomMessage?: boolean;
 }
-const CalendarList: React.FC<CalendarListProps> = ({ calendars, page }) => (
-  <div css={containerStyles}>
-    <Card>
-      <div css={headerStyles}>
-        <Headline3>
-          Subscribe to{' '}
-          {page === 'group'
-            ? "this group's"
-            : page === 'event'
-            ? "this event's"
-            : 'Groups on'}{' '}
-          calendar
-        </Headline3>
-      </div>
-      {page === 'calendar' && (
+const CalendarList: FC<CalendarListProps> = ({
+  calendars,
+  page,
+  showBottomMessage = true,
+}) => {
+  const truncateFrom = 5;
+  const [showMore, setShowMore] = useState(false);
+  const displayShowMoreButton = calendars.length > truncateFrom;
+
+  return (
+    <div css={containerStyles}>
+      <Card padding={false}>
+        <div css={headerStyles}>
+          <Headline3 noMargin>
+            Subscribe to{' '}
+            {page === 'group'
```

Comment:
I think this would be better done with a Record object. There are too many ternary operators to follow.

`
{
'group' : {
	headline: 'aaa',
	description: 'bbb',
	}
}

`